lib/screens/authentication/recover/recover_account_details/interactor/viewmodels/recover_account_details_state.dart

```diff
@@ -4,35 +4,52 @@ class RecoverAccountDetailsState extends Equatable {
   final PageState pageState;
   final String userAccount;
   final PageCommand? pageCommand;
+  final Uri? linkToActivateGuardians;
+  final int totalGuardiansCount;
```

Comment:
Would be better to call this threshold, since that's the name it has on the backend